plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/StartAgentJob.java

```diff
@@ -122,7 +122,7 @@ private void initialize(CodyAgentServer server, Path workspaceRoot)
     // we need to wait for enterprise customers to upgrade to a new version that includes the fix
     // from this PR here https://github.com/sourcegraph/sourcegraph/pull/63855.
     clientInfo.name = "jetbrains";
-    clientInfo.version = "5.5.20-eclipse"; // Needs to be greater than 5.5.8
+    clientInfo.version = "5.5.21-eclipse"; // Needs to be greater than 5.5.8
```

Comment:
This is confusing but we need a version above 5.5.10 to get past checks on the server. We should probably just use the same version scheme for the eclipse plugin instead of 0.x. FYI @jamesmcnamara